spec/unit/installer/analyzer_spec.rb

```diff
@@ -87,16 +90,18 @@ module Pod
 
       #--------------------------------------#
 
-      it 'generates the libraries which represent the target definitions' do
+      it 'generates the model to represent the target definitions' do
         target = @analyzer.analyze.targets.first
         target.pod_targets.map(&:name).sort.should == [
-          'Pods-JSONKit',
-          'Pods-AFNetworking',
-          'Pods-SVPullToRefresh',
+          'JSONKit',
+          'AFNetworking',
+          'SVPullToRefresh',
           'Pods-libextobjc',
         ].sort
         target.support_files_dir.should == config.sandbox.target_support_files_dir('Pods')
 
+        target.pod_targets.map(&:archs).uniq.should == [[]]
```

Comment:
Because the analyzer should recognize that the user targets do not overwrite this build setting and won't overwrite it in the pod targets. Before it was nil, a string or an array.